include/date_time.h

```diff
@@ -70,7 +70,8 @@ int date_time_set(const struct tm *new_date_time);
  *
  *  @return 0        If the operation was successful.
  *  @return -ENODATA If the library does not have a valid date time UTC.
- *  @return -EINVAL  If the passing variable is too large or already converted.
+ *  @return -EINVAL  If the passed in pointer is NULL, dereferenced value is too large,
```

Comment:
Should probably use `retval` for these, but since the rest of the API is documented with use of `return` I decided to align with that. Should be changed for all instances.

Reply:
Yeah, let's keep as it is, but indeed `retval` should be used instead. Would be good to fix that at some point.